File: JavaScript_Courses/JavaScript_Basics/Preparation/02_Conditional_Statements_Exercise/05_GodzillaVsKong.js
function godzillaVsKong(input) {
    let budget = Number(input[0]);
    let statists = Number(input[1]);
    let clothesPrice = Number(input[2]);

    let decor = budget * 0.10;
    let clothesAll = statists * clothesPrice;

    if (statists > 150) {
        clothesAll *= 0.90;
    }

    let expenses = decor + clothesAll;

    if (expenses > budget) {
        console.log('Not enough money!');
        console.log(`Wingard needs ${(expenses - budget).toFixed(2)} leva more.`);
    } else {
        console.log('Action!');
        console.log(`Wingard starts filming with ${(budget - expenses).toFixed(2)} leva left.`);
    }
}


// godzillaVsKong(["15437.62", "186", "57.99"]);

File: Python_Courses/Python_Basics/Python_PB_2023_10/02_Conditional_Statements_Exercise/05_Godzilla_vs_Kong.py
budget = float(input())
extras = int(input())
clothing_for_extra = float(input())

decor = budget * 0.10
if extras > 150:
    clothing_for_extra = clothing_for_extra * 0.90

money_needed = decor + extras * clothing_for_extra

if budget >= money_needed:
    print(f"Action!\nWingard starts filming with {budget - money_needed :.2f} leva left.")
else:
    print(f"Not enough money!\nWingard needs {money_needed - budget :.2f} leva more.")
